Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -40,6 +40,7 @@ def setUp(self):
             self.resource_url = config["resource_url"]
             self.scope = config["scope"]
             self.subscription_id = config["subscription_id"]
+            self.container_name = config["container_name"]
```

Comment:
Please add this to cohesity.json and provide the expected default value. Also, could you rename the config variable to blob_container_name?

Reply:
the az cli parameter is --container-name, so, better to keep container_name.

Reply:
done

Reply:
I already gave the explanation for 'scope' parameter in the last PR which was not resolved.

The name used inside cohesity.json doesn't have to be the same name inside the function here. For someone opening cohesity.json and looking at container_name will have no idea whether it's a storage container or a docker container. In az cli, it's obvious from the context that we're performing a storage account query, hence container-name makes sense there.

Reply:
container_name is still not renamed.